{{cookiecutter.github_repo}}/{{cookiecutter.python_package_namespace}}/{{cookiecutter.python_subpackage}}/__init__.py

```diff
@@ -0,0 +1 @@
+default_app_config = '{{cookiecutter.python_package_namespace}}.{{cookiecutter.python_subpackage}}.apps.{{ cookiecutter.python_subpackage|title|replace(' ','') }}Config'
```

Comment:
What does this line do? I have seen similar lines added to __init__.py's before but I have never understood what exactly it accomplishes.

Reply:
@DingoEatingFuzz had added it while we were troubleshooting, but I think we may actually be able to drop it as per:
https://docs.djangoproject.com/en/2.2/ref/applications/#configuring-applications
Appears deprecated for current Django versions

Reply:
Yep, this was me experimenting. This had nothing to do with what ended up being that actual issue.

Reply:
Ok got it. I thought it actually did something useful that I didn't know about yet. :)

Reply:
removed the line